ui/src/app/pages/profile/data-use-agreement.tsx

```diff
@@ -41,262 +33,27 @@ const styles = reactStyles({
   }
 });
 
-const SecondHeader = (props) => {
-  return <h2 style={{...styles.h2, ...props.style}}>{props.children}</h2>;
+export const SecondHeader = (props) => {
+  return <h2 style={{...dataUseAgreementStyles.h2, ...props.style}}>{props.children}</h2>;
 };
 
-const indentedListStyles = {
+export const indentedListStyles = {
```

Comment:
I'd maybe move these to the other file, since they are used there and not here. (Sorry for getting into code styling)